x/intrarelayer/client/cli/tx.go

```diff
@@ -0,0 +1,429 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	ethermint "github.com/tharsis/ethermint/types"
+	"github.com/tharsis/evmos/x/intrarelayer/types"
+)
+
+// NewTxCmd returns a root CLI command handler for certain modules/intrarelayer transaction commands.
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "intrarelayer subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	txCmd.AddCommand(
+		NewConvertCoinCmd(),
+		NewConvertERC20Cmd(),
+	)
```

Comment:
Should these be added?
NewRegisterCoinProposalCmd, 
NewRegisterERC20ProposalCmd, 
NewToggleTokenRelayProposalCmd, 
NewUpdateTokenPairERC20ProposalCmd

Reply:
you can use `evmosd tx submit-proposal`

Reply:
it was tested with `evmosd tx gov submit-proposal` and it's working fine